docs/concepts/rest_apis.rst

```diff
@@ -0,0 +1,32 @@
+edx-platform REST API Concepts
+##############################
+
+APIs in the edx-platform fall into one of two categories.  **Personal APIs**
```

Comment:
This is a general comment, but headers and lists can make this much quicker to digest. As an example, this paragraph is calling out to be a numbered list. Same below. Subheaders may also help you organize and clarify what is being presented.

See https://github.com/openedx/docs.openedx.org/issues/338 for some good resources and tips.

I feel like an updated version would both be simpler to review and be simpler for your intended audience to digest, so I'll wait to see what you decide.